lib/PerlIO.pm

```diff
@@ -220,10 +332,9 @@ but then enable UTF-8 translation.
 
 =item :pop
 
-A pseudo-layer that removes the top-most layer. Gives Perl code a
```

Comment:
It's still true that it doesn't undo the effects of pseudo-layers or flags, even if :utf8 is no longer an appropriate example. Also seems to have lost the minor typographic fixes from https://github.com/Perl/perl5/pull/17630.

Reply:
Reverted, and now refers to `:bytes`  instead of `:utf8`